src/components/structures/SpaceRoomView.tsx

```diff
@@ -672,25 +671,11 @@ export default class SpaceRoomView extends React.PureComponent<IProps, IState> {
 
         if (payload.action !== Action.ViewUser && payload.action !== "view_3pid_invite") return;
 
-        if (payload.action === Action.ViewUser && payload.member) {
-            const spaceMemberInfoCard: IRightPanelCard = {
-                phase: RightPanelPhases.SpaceMemberInfo,
-                state: { spaceId: this.props.space.roomId, member: payload.member },
-            };
-            if (payload.push) {
-                RightPanelStore.instance.pushCard(spaceMemberInfoCard);
-            } else {
-                RightPanelStore.instance.setCards([
-                    { phase: RightPanelPhases.SpaceMemberList, state: { spaceId: this.props.space.roomId } },
-                    spaceMemberInfoCard,
-                ]);
-            }
-        } else if (payload.action === "view_3pid_invite" && payload.event) {
-            RightPanelStore.instance.setCard({
-                phase: RightPanelPhases.Space3pidMemberInfo,
-                state: { spaceId: this.props.space.roomId, memberInfoEvent: payload.event },
-            });
-        } else {
+        /**
+         * The rest of the `ViewUser` and `view_3pid_invite` exists in the `<RoomView />`
+         * component. This deals specifically with showing the space members list
+         */
+        if (!payload.member && !payload.event) {
```

Comment:
I think it's not entirely clear what is going on here. Can we combine it with the if statement above, and do something like:

```suggestion
        if (
            (payload.action === Action.ViewUser && !payload.member) ||
            (payload.action === "view_3pid_invite" && payload.event === null)
        ) {
```

Reply:
fixed in https://github.com/matrix-org/matrix-react-sdk/pull/11495/commits/89fa7b4b396e83cb8204ac661a72fe02faac6851